draft-ZIP-0227.rst

```diff
@@ -0,0 +1,184 @@
+- `issuerID` is the public key of the issuer, used to verify the signature on the transaction and is defined as `ik` (issuer key or issuer verification key):
+
+
+:math:`\mathsf{ik := SpendAuthSig^{Orchard}.DerivePublic(isk)}`
+
+
+which uses the same signature as the spend authorization signature, where `isk` is the issuer secret key. The verification key is derived from `isk`. In order to differentiate the `isk` and `ik` keys from the actual spend authorization key, we derive another branch of keys from the main spending key `sk` as such:
+
+:math:`\mathsf{isk := ToScalar^{Orchard}(︀ PRF^{expand}_{sk} ([10])}`
+
+This allows the issuer to use the same wallet it usually uses to transfer assets, while keeping a disconnect from the other keys. It provides further anonymity and the ability to delegate issuance of an asset (or in the future, generate a multi-signature protocol) while the rest of the keys in the wallet safe.
+
+Then we let `issuanceInfo := (assetID, finiteSupply)`, where 
+
+- `assetID` is as defined above.
+- `finiteSupply` is a boolean variable that defines whether this issuance transaction of the specific asset type is the final issuance of tokens or not. Once `finiteSupply` is set, it cannot be unset. This allows expanding the functionality of the issuance mechanism:
+    - Provides `assetID` revocation in case of compromise (issue last issuance with 0 token value)
+    - Supports NFT issuance (where the first issuance with value of 1 is also the last)
+
+
+Issuance Protocol
+=================
+
+The issuance protocol allows for a single issuance to be sent to many receivers, as the issuerID does not have to match the address ownership of the notes output. Furthermore, every transaction can contain many issuance instances. The design presented in this ZIP enables several use cases for issuance of shielded assets.
+
+- The issuer knows in advance the receivers of the issued asset.
+- The asset is of non-fungible type, where each asset type can be made part of a single “series”
+- The supply of the asset is limited or not
+- The assets can be wrapped versions of assets in other chains (as long as there is a bridge that supports it)
+
+The protocol is as follows:
+
+- For each asset type issued, generate a sequence of output notes, each with the following fields:
+    - ZSA output note :math:`\mathsf{ note =(d, pkd, v, \rho, \psi, \mathsf{type}_\mathbb{P}, rcm)}`, where :math:`\mathsf{type}_\mathbb{P} := \mathsf{GroupHash^{\mathbb{P}}}\mathsf{(assetID)}`
+- Generate commitment of the note. `cm` as
+
+:math:`cm = \mathsf{NoteCommit^{ZSA}_{rcm}}(\mathsf{repr\mathbb{_P}(g_d)}, \mathsf{repr\mathbb{_P}(pk_d)}, v, \rho, \psi,\mathsf{type}_\mathbb{P})`
+
+- Sign the issuance note with the `issuerID` as the signing key, using RedPallas as its signature scheme, on the `SIGHASH` of the transaction. Note that the `SIGHASH` will change as we include a new bundle in the Zcash transaction to enable this issuance mechanism on chain.
```

Comment:
Seems we need to define the new bundle so we know the modification of the transaction

Reply:
Done. See the new PR.